proto/ligato/vpp/wireguard/wireguard.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package ligato.vpp.wireguard;
+
+option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/wireguard;vpp_wg";
+
+message Peer {
+  // Public-key base64
+  string public_key = 1;
+
+  // Peer UDP port
+  uint32 port = 2;
+
+  // Keepalive interval (sec)
+  uint32 persistent_keepalive = 3;
+
+  //  Table ID
+  uint32 table_id = 4;
+
+  // Endpoint IP
+  string endpoint = 5;
+
+  // Wireguard interface name
+  string interface = 6;
+
+  // Flags
+  // WIREGUARD_PEER_STATUS_DEAD = 0x1
+  uint32 flags = 7;
+
+  // Allowed IPs count
+  uint32 n_allowed_ip = 8;
```

Comment:
no need for this field - this is not as low-level as VPP binary API, you can do `len(AllowedIps)`